build.sbt

```diff
@@ -208,7 +208,7 @@ lazy val circeCrossModules = Seq[(Project, Project)](
 )
 
 lazy val circeJsModules = Seq[Project](scalajs, scalajsJavaTimeTest)
-lazy val circeJvmModules = Seq[Project](benchmark, jawn)
+lazy val circeJvmModules = Seq[Project](benchmark, jawn, extras)
```

Comment:
You shouldn't need this, since this module is cross-built (sorry the build setup is unclear here).

Reply:
So it should just be `lazy val circeJvmModules = Seq[Project](benchmark, jawn)`?

Reply:
https://github.com/circe/circe/pull/1436/commits/882facef613cdc93d2366f8727b52952534ea8d3

> since this module is cross-built 

`this module` = `extras`?